pom.xml

```diff
@@ -79,7 +79,8 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <version>1.580.1</version>
+      <!--to have access to User.getById-->
+      <version>1.651.2</version>
```

Comment:
In such case we will need to bump the default baseline in Plugin POM. BTW, Plugin POM 3.x already did it: https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#30

Reply:
IIUC the changelog, it's safe to bump the baseline to 2.7.x at least. CC @batmat

Reply:
`plugin-pom` 3.0 did not really drop support for Jenkins 1; it just stopped enforcing a check on usages on Servlet APIs. Switching _this_ dependency would really make it impossible to use JTH, and thus a `plugin-pom` incorporating it (without overriding JTH version), on a plugin with an older baseline.

I am not opposed to doing that; maybe now is the time. https://github.com/jenkinsci/plugin-pom/pull/83#pullrequestreview-72362389